nuxt.config.ts

```diff
@@ -69,6 +69,7 @@ const config: Configuration = {
   plugins: [
     '~/plugins/router-hooks.ts',
     '~/plugins/directives.ts',
+    './plugins/carbon.ts',
```

Comment:
Please, change the initial period `.` with the tilde `~`.

Reply:
True, I didn't even see this!

Reply:
Fixed! 👍